Contents/mods/CzechTranslation/42/media/lua/shared/Translate/CS/ItemName_CS.txt

```diff
@@ -1,64 +1,83 @@
-    ItemName_Base.45Clip = "Zásobník .45 Auto",
-    ItemName_Base.556Box = "Krabička s náboji 5.56mm",
-    ItemName_Base.556Bullets = "Náboj 5.56 mm",
-    ItemName_Base.556Clip = "Zásobník 5.56 mm",
-    ItemName_Base.9mmBulletsMold = "Forma na náboje 9 mm",
-    ItemName_Base.9mmClip = "Zásobník 9 mm",
+    ItemName_Base.308Carton = "Karton nábojů .308",
+    ItemName_Base.308Clip = "Zásobník MSR788",
+    ItemName_Base.44Clip = "Zásobník D-E",
+    ItemName_Base.45Clip = "Zásobník M1911",
+    ItemName_Base.556Box = "Krabička nábojů 5.56mm",
+    ItemName_Base.556Bullets = "Náboj 5.56mm",
+    ItemName_Base.556Carton = "Karton nábojů 5.56",
+    ItemName_Base.556Clip = "Zásobník M16",
+    ItemName_Base.9mmBulletsMold = "Forma na náboje 9mm",
+    ItemName_Base.9mmClip = "Zásobník M9",
     ItemName_Base.Acorn = "Žalud",
+    ItemName_Base.AdhesiveBandageBox = "Krabička náplastí",
     ItemName_Base.Aerosolbomb = "Aerosolová bomba",
-    ItemName_Base.AerosolbombRemote = "Aerosolová bomba s dálkovým ovládáním",
-    ItemName_Base.AerosolbombSensorV1 = "Aerosolová bomba se senzorem",
-    ItemName_Base.AerosolbombSensorV2 = "Aerosolová bomba se senzorem",
-    ItemName_Base.AerosolbombSensorV3 = "Aerosolová bomba se senzorem",
+    ItemName_Base.AerosolbombRemote = "Aerosolová bomba s dálkovým odpalovaním",
+    ItemName_Base.AerosolbombSensorV1 = "Aerosolová bomba se senzorem (krátký dosah)",
+    ItemName_Base.AerosolbombSensorV2 = "Aerosolová bomba se senzorem (střední dosah)",
+    ItemName_Base.AerosolbombSensorV3 = "Aerosolová bomba se senzorem (dlouhý dosah)",
     ItemName_Base.AerosolbombTriggered = "Aerosolová bomba s časovačem",
     ItemName_Base.AlarmClock = "Budík",
     ItemName_Base.AlarmClock2 = "Budík",
-    ItemName_Base.AlcoholBandage = "Sterilní obvaz",
-    ItemName_Base.AlcoholRippedSheets = "Sterilní hadr",
-    ItemName_Base.AlcoholWipes = "Ubrousky s alkoholem",
-    ItemName_Base.AlcoholedCottonBalls = "Vata s alkoholem",
-    ItemName_Base.Allsorts = "Různé",
-    ItemName_Base.Aluminum = "Alobal",
-    ItemName_Base.AmericanLadyCaterpillar = "Caterpillar",
-    ItemName_Base.AmmoStrap_Bullets = "Popruh na náboje",
-    ItemName_Base.AmmoStrap_Shells = "Popruh na brokovnicové náboje",
-    ItemName_Base.AmmoStraps = "Popruh na munici",
+    ItemName_Base.AlcoholBandage = "Obvaz (Sterilizovaný)",
+    ItemName_Base.AlcoholRippedSheets = "Hadr (Sterilizovaný)",
+    ItemName_Base.AlcoholWipes = "Alkoholové ubrousky",
+    ItemName_Base.AlcoholedCottonBalls = "Vatové tampony namočené v alkoholu",
+    ItemName_Base.Allsorts = "Lékořicové bonbony",
+    ItemName_Base.Aluminum = "Hliníková fólie",
+    ItemName_Base.AmericanLadyCaterpillar = "Housenka",
+    ItemName_Base.AmmoStrap_Bullets = "Bandalír na náboje",
+    ItemName_Base.AmmoStrap_Bullets_223 = "Bandalír na náboje",
+    ItemName_Base.AmmoStrap_Bullets_308 = "Bandalír na náboje",
+    ItemName_Base.AmmoStrap_Shells = "Bandalír na brokové náboje",
```

Comment:
Popruh na brokovnicové náboje

Opravdu to jsou popruhy. Bandalír je na středověký meč 

https://armyman.cz/cs/prisl-pro-brokovnice/635-ramwear-shotgun-50round-killer-69-popruh-na-brokovnicove-naboje.html

Zbytek zitra